homeassistant/components/tado/climate.py

```diff
@@ -256,6 +263,7 @@ def __init__(
         self._attr_fan_modes = supported_fan_modes
         self._attr_supported_features = support_flags
 
+        # Why are they here? Part of generic thermostat it seems? Need to be set?
```

Comment:
Why is this question here?

Reply:
They seem unused. Is needed for all climate controls? I simply don't know why it's there at this point.

Reply:
Then please remove them if they are not used in the parent classes.